programs/pkey/pk_encrypt.c

```diff
@@ -135,13 +136,17 @@ int main( int argc, char *argv[] )
      */
     if( ( f = fopen( "result-enc.txt", "wb+" ) ) == NULL )
     {
-        mbedtls_printf( " failed\n  ! Could not create %s\n\n", "result-enc.txt" );
+        mbedtls_printf( " failed\n  ! Could not create %s\n\n",
```

Comment:
I know it's in the original, but if you already modified this line, hwy are we have string formatting with a literal string?
Wouldn't it be more readable: `mbedtls_printf( " failed\n  ! Could not create result-enc.txt\n\n" );` ?

Reply:
It's slightly more easier to change this way I guess.